action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -406,6 +426,25 @@ func (stateDB *StateDBAdapter) Suicide(evmAddr common.Address) bool {
 		stateDB.logError(err)
 		return false
 	}
+	// before calling Suicide, EVM will transfer the contract's balance to beneficiary
+	// need to create a transaction log on successful suicide
+	if stateDB.lastAddBalanceAmount.Cmp(actBalance) == 0 {
+		if len(stateDB.lastAddBalanceAmount.Bytes()) > 0 {
+			from, _ := address.FromBytes(evmAddr[:])
+			stateDB.addTransactionLogs(&action.TransactionLog{
```

Comment:
is not a hard-fork?

Reply:
We believe it is not, because transaction logs are not included in delta. @dustinxie may confirm this.